CP-8644 github action: repurpose script to run erb_lint gem

jwshinx opened the pull request:
CP-8644

this is a redo of the CP `erb-lint-action` repo.

first...
- i forked at [rubocop-action here](https://github.com/CareerPlug/rubocop-action/commits/master/)
- i removed this repo's history
- pushed the first commit

this pr makes a few tweaks to repurpose the script to run the `erb_lint` gem.

## considerations
- once merged, ill create a release/tag number, and do a ats pull request test.
- once we prove out, i will circle back and clean up, readme, default rubocop reminents, etc.

## references
https://github.com/Shopify/erb_lint
https://www.rubydoc.info/gems/erb_lint/0.9.0/index

cc @igray

igray reviewed (approved):
Looks great! This really shows how it compares to the rubocop action.

jwshinx committed:
repurpose script to run erb_lint gem

https://github.com/Shopify/erb_lint
https://www.rubydoc.info/gems/erb_lint/0.9.0/index

igray reviewed (approved):
This is great. Thanks for reforking. It's probably unnecessary to change the exit code, but no matter. It's good as-is.
